GrandCentralBoardTests/Widgets/Harvest/CircleChartViewTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  CircleChartViewTests.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 17.05.2016.
+//  Copyright © 2016 Macoscope. All rights reserved.
+//
+
+import FBSnapshotTestCase
+@testable import GrandCentralBoard
+
+
+final class CircleChartViewTests: FBSnapshotTestCase {
+
+    func testEvenDistribution() {
+        let items = [
+            CircleChartItem(color: .redColor(), ratio: 1.0 / 3),
```

Comment:
There should be different colors. Please consider a `cherry-pick` of https://github.com/macoscope/GrandCentralBoard/pull/131/commits/bb447fb8d4fe700cc7b85610a12bff72a322553f